composer.json

```diff
@@ -80,25 +80,31 @@
         "sort-packages": true
     },
     "scripts": {
+        "post-autoload-dump": [
+            "@install-tools"
+        ],
         "cs:check": "@php php-cs-fixer fix --dry-run --diff",
         "cs:fix": "@php php-cs-fixer fix",
-        "dev-tools:check": [
-            "./dev-tools/check_file_permissions.sh",
-            "./dev-tools/check_trailing_spaces.sh"
-        ],
-        "dev-tools:docs": "@php dev-tools/doc.php",
-        "dev-tools:install": "@composer -d dev-tools install",
-        "docs": "@dev-tools:docs",
+        "docs": "@php dev-tools/doc.php",
+        "install-tools": "@composer -d dev-tools install",
         "mess-detector": "@php dev-tools/vendor/bin/phpmd . ansi phpmd.xml",
-        "phpstan": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse",
-        "phpstan:baseline": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse --generate-baseline=./dev-tools/phpstan/baseline.php",
+        "normalize": [
```

Comment:
composer:normalize?

we can normalize many things, eg line ending, CS, file chods, ...

Reply:
- `composer composer:normalize` does not look good and does not make much sense
- it's simulating regular usage of that plugin (just look what it does internally). When you have it installed in main `composer.json` you use it like this 🙂